examples/voting/voting.py

```diff
@@ -107,7 +110,7 @@ def bootstrap(self, fund_min_bal_req: PaymentTransaction) -> None:
         assert (
             fund_min_bal_req.amount == min_balance_req
         ), "Payment must be for the exact min balance requirement"
-        assert Box.create(TALLY_BOX_KEY, tally_box_size)
+        self.tallies.value = bzero(tally_box_size)
```

Comment:
What’s bzero?

Reply:
creates a byte array of length N (all zeros)